tools/topology/CMakeLists.txt

```diff
@@ -122,6 +122,7 @@ set(TPLGS
 	"sof-tgl-max98357a-rt5682\;sof-tgl-max98357a-rt5682\;-DCODEC=MAX98357A\;-DFMT=s16le"
```

Comment:
@rosschisholm-xperi would be nice to add more details in the commit message for the reader to better understand the context. For example what are Xperi tunings more exactly.